infoblox_client/feature.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2015 Infoblox Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import re
+import six
+
+from infoblox_client import exceptions as ib_ex
+
+FEATURE_VERSIONS = [
+    {'feature': 'create_ea_def', 'min_ver': '2.2'},
```

Comment:
I think feature format can be simplified to:

    FEATURE_VERSIONS = {
        'create_ea_def' : '2,2',
        'cloud_api': '2.0'}

Feature name is unique so can be used as key in a dict, and I believe the only argument is needed is minimal version.
In this case comment describing FEATURE_VERSIONS format is needed.